src/sophia/utility.f

```diff
@@ -0,0 +1,69 @@
+      SUBROUTINE TOEVT()
+C************************************************************************
+C
```

Comment:
The file should be called impy_sophia.f in line with what is there for all other generators EXCEPT SIBYLL :) It's an historical oversight and need to be corrected there, too.

Reply:
The file is renamed utility.f -> impy_sophia.f